### DISCUSSION TITLE
Why "https://search.maven.org/" is required instead of  "https://repo1.maven.org/maven2" ?

### DISCUSSION THREAD

**Discussion by tvallin** [Homebrew/discussions#4276] [Writing Formulae/Casks]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

`brew audit --strict` output : 
`https://repo1.maven.org/maven2/...` should be `https://search.maven.org/remotecontent?filepath=...`

My concern is about this topic: https://central.sonatype.org/faq/what-happened-to-search-maven-org/

I would like to know the reason about this decision and if there is any plan on allowing `https://repo1.maven.org/maven2/` url ?

Thanks!

**Comment by tvallin** [comment]
@SMillerDev any short explanation behind this decision ? 

Thanks!

**Reply by SMillerDev** [reply]
No
